pkg/cluster/disableupdates.go

```diff
@@ -1,26 +0,0 @@
-package cluster
-
-// Copyright (c) Microsoft Corporation.
-// Licensed under the Apache License 2.0.
-
-import (
-	"context"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/util/retry"
-)
-
-func (i *manager) disableUpdates(ctx context.Context) error {
```

Comment:
I'd love to get rid of this but I don't think we can.  Also note: there's no point adding the customDNS check if we're just going to have all the clusters on the stable channel...

Reply:
so are we intentionally leaving cluster behind with this? Based on the new policy?

Reply:
I was thinking about this more (needed more coffee).
 I disagree with this, but I disagree with the main reason we doing all this so this is wider disagreement :D 

Should we follow-up with this on MSFT scrum? as this does not sound right to me :/